frontend/pages/[resourceType]/[id].tsx

```diff
@@ -3,13 +3,31 @@ import { Divider, Group, Stack, Tabs, Text } from '@mantine/core';
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next';
 import { FhirArtifact } from '@/util/types/fhir';
 import BackButton from '../../components/BackButton';
+import { useEffect, useState } from 'react';
+import PrismRenderer from '../../prismLanguages/prismCQL';
 
 /**
  * Component which displays the JSON/ELM/CQL/narrative content of an individual resource using
  * Mantine tabs
  * @returns JSON/ELM/CQL/narrative content of the individual resource in a Prism component
  */
 export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType<typeof getServerSideProps>) {
+  const [decodedCql, setDecodedCql] = useState<string | null>(null);
+
+  // Overwrite Prism with our custom Prism that includes CQL as a language
+  useEffect(() => {
+    (window as any).Prism = PrismRenderer;
```

Comment:
If you go with my comment in the helper, you'll have to set the `.languages.cql` property in here instead